android/java/org/chromium/chrome/browser/playlist/PlaylistHostActivity.java

```diff
@@ -235,15 +233,18 @@ public void finishNativeInitialization() {
                             if (mPlaylistService == null) {
                                 return;
                             }
-                            PlaylistOptionsEnum option = playlistOptionsModel.getOptionType();
-                            if (option == PlaylistOptionsEnum.DELETE_PLAYLIST) {
+                            PlaylistModel.PlaylistOptionsEnum option =
```

Comment:
why did this change from `PlaylistModelEnum` -> `PlaylistModel.PlaylistOptionsEnum`?

Reply:
We had to update as the current chromium compiler for kotlin doesn't recognize enums as classes. So I had to include it as a part of a kotlin class.